### PR TITLE
Updating SDK

### PR DESCRIPTION
**Pull request by sismanis**
Smoke tested with setting this as the new local maven repo

### CODE REVIEW

File: devicesetup/devicesetup.iml [hunk 1]
```diff
@@ -46,53 +45,55 @@
...
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/res" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/rs" />
+      <excludeFolder url="file://$MODULE_DIR$/build/intermediates/shaders" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/symbols" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/transforms" />
       <excludeFolder url="file://$MODULE_DIR$/build/outputs" />
       <excludeFolder url="file://$MODULE_DIR$/build/tmp" />
     </content>
-    <orderEntry type="jdk" jdkName="Android API 23 Platform" jdkType="Android SDK" />
+    <orderEntry type="jdk" jdkName="Android API 25 Platform (1)" jdkType="Android SDK" />
```
Comment (philipmeier) [inline devicesetup/devicesetup.iml:96]: whats the "(1)" ?

File: devicesetup/devicesetup.iml [hunk 2]
```diff
@@ -46,53 +45,55 @@
...
       <sourceFolder url="file://$MODULE_DIR$/src/androidTest/rs" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/androidTest/shaders" isTestSource="true" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/res" type="java-test-resource" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/resources" type="java-test-resource" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/assets" type="java-test-resource" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/aidl" isTestSource="true" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/java" isTestSource="true" />
-      <sourceFolder url="file://$MODULE_DIR$/src/test/jni" isTestSource="true" />
       <sourceFolder url="file://$MODULE_DIR$/src/test/rs" isTestSource="true" />
+      <sourceFolder url="file://$MODULE_DIR$/src/test/shaders" isTestSource="true" />
```
Comment (philipmeier) [inline devicesetup/devicesetup.iml:77]: what are all of these "shaders" changes?

Comment (philipmeier) [reply devicesetup/devicesetup.iml:77]: i suspect the is openGL stuff.  Is this the consequence of 23-->25, and how did it get here?

**Review by philipmeier — CHANGES_REQUESTED** [review]
I am cool with these changes, after answering Qs and confirming that the build is reproducible on another laptop.  I think that maybe on Monday, Csaba and I should sit down with you and confirm that we can build.   I suspect most of this will be you helping us out with standard setup, and then the last bit is confirming the maven dependencies work the same.

**Comment by sismanis** [comment]
I will be in later today to help you guys get it compiling. These changes were auto-generated when I upgraded the gradle file. There might be an argument for including this file in .gitignore. Not 100% sure on that though. We can discuss today while I am in.

**Pull request closed without merging**